exercises/variable-length-quantity/canonical-data.json

```diff
@@ -0,0 +1,147 @@
+{
+  "#": [
+    "JSON doesn't allow hexadecimal literals.",
+    "All numbers are given as decimal literals instead.",
+    "It is highly recommended that your track's test generator display all numbers as hexadecimal literals."
```

Comment:
I'm aware not all tracks have them, but maybe this can be considered as a gentle nudge to get tracks to consider the practice... so this description was intentionally worded provocatively in this way.

I hope this doesn't make this file too inconvenient. I would be sad to store the string representations of the hexadecimal literals.